web/modules/custom/dpl_dashboard/src/Plugin/Block/DashboardBlock.php

```diff
@@ -102,11 +102,14 @@ public function build(): array {
       'ereolen-my-page-url' => $generalSettings->get('ereolen_my_page_url'),
 
       // Texts.
+      'reservation-details-others-in-queue-text' => $this->t('Others are queueing for this material', [], ['context' => 'Dashboard']),
```

Comment:
Please double check with the search app strings if some trnaslations could be resused. Stupid to translate it twice. Or what do you think?